sw_utils/tests/factories.py

```diff
@@ -36,6 +36,10 @@ def eth_public_key(self) -> str:
         # 48 bytes
         return '0x' + ''.join(random.choices('abcdef' + string.digits, k=96))
 
+    def eth_uncompressed_public_key(self) -> str:
+        # 128 bytes
```

Comment:
64 bytes
1 byte = 2 hex symbols

Reply:
Also I suggest `ecies` prefix instead of `eth`. Because `eth` was used for BLS pub keys (signatures).